.docker/api/data/scholarly-articles/article1/1.xml

```diff
@@ -1,12 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<item xmlns="http://libero.pub">
+<item xmlns="http://libero.pub" xmlns:jats="http://jats.nlm.nih.gov">
 
-    <front xml:lang="en">
+    <meta>
 
         <id>article1</id>
-        <title>Article 1</title>
```

Comment:
not yet, but still makes me think some version of the title (maybe a processed one, removing italic/bold/secondary formatting) should appear in `<meta>` to simplify listings from other services